Tools/cases_generator/generators_common.py

```diff
@@ -652,6 +654,19 @@ def emit_tokens(
             raise analysis_error(ex.args[0], rbrace) from None
         return storage
 
+    def emit_tokens_simple(
+        self,
+        tokens: list[Token]
+    ) -> None:
+        tkn_iter = TokenIterator(tokens)
+        self.out.start_line()
+        for tkn in tkn_iter:
+            if tkn.text in self._replacers:
+                self._replacers[tkn.text](tkn, tkn_iter, None, None, None) # type: ignore[arg-type]
```

Comment:
Is the `type: ignore` needed?

Reply:
Yes, because we specify the the replacers don't take None, but in this case they do. Fixing the type annotation would need invasive changes to the rest of the code to assert not None.

Reply:
```suggestion
                storage = Storage(Stack(), [],[], [])
                self._replacers[tkn.text](tkn, tkn_iter, label, storage, None)
```

AFAICT the only use of the `uop` is to gets its name and instruction size.
Change the `Uop` to `Label | Uop` in the replacement function type annotation and make sure `Label` has `instruction_size = None` and this should type check.

Reply:
Done.